diff --git a/node_modules/@html-eslint/eslint-plugin/lib/rules/element-newline.js b/node_modules/@html-eslint/eslint-plugin/lib/rules/element-newline.js
index 2edb79f..ac7dfed 100644
--- a/node_modules/@html-eslint/eslint-plugin/lib/rules/element-newline.js
+++ b/node_modules/@html-eslint/eslint-plugin/lib/rules/element-newline.js
@@ -132,7 +132,7 @@ module.exports = {
         }
         childrenToCheck.push(child);
       }
-      return childrenToCheck.filter((child) => !isEmptyText(child));
+      return childrenToCheck.filter((child) => !isEmptyText(child) && !isComment(child));
     }
 
     /**
